dashboard/settings.py

```diff
@@ -291,5 +291,12 @@
     AUTHENTICATION_BACKENDS += ('django.contrib.auth.backends.ModelBackend',)
     LOGIN_REDIRECT_URL = '/'
     
+# This is fixed from UDW
+UDW_ID_PREFIX = config("UDW_ID_PREFIX", default=17700000000, cast=int)
+
+# TODO: Select a default course for user from database, just here so things don't all break
+# This will probably need to move to views.py
+DEFAULT_COURSE_ID = config("CANVAS_COURSE_IDS", default=0, cast=int)
```

Comment:
I would think the CANVAS_COURSE_IDS consists a course id array, unlike the DEFAULT_COURSE_ID is just one single course id.

Reply:
Yeah, at the moment this was only singular (even though the variable name says IDS)

I think we should remove ID's from this completely and store it in the database. I can work on this but the 'default' for a user is going to be different than what this array was. I just didn't want it to completely break on the dev/prod server(s)